client/app/models/contact.coffee

```diff
@@ -66,13 +66,16 @@ module.exports = class Contact extends Backbone.Model
                         @fetch()
 
                     error: options.error
+        super
+
 
+    validate: (model, options) ->
```

Comment:
According to BB's doc, first arg is `attrs`, not the model itself, and is a JSON representation of model's attributes. I suggest to rename it as `attrs` to avoid confusion